ts/input/tex/TexParser.ts

```diff
@@ -572,7 +572,8 @@ export default class TexParser {
       }
       return;
     }
-    if (node.attributes.get(TexConstant.Attr.LATEX) === '_' && str !== '_') {
+    if (node.attributes.get(TexConstant.Attr.LATEX) === '_' &&
+      str !== '_' && str !== '\\_') {
```

Comment:
I think you need a similar fix for `^` in line 561 above, in case anyone creates a macro `\^` similar to the `\_` one.

Reply:
At the moment that can't really happen as latex computation happens after the macro replacement.
But I am not really happy with that anyway, so I've added the case.  
PTAL.